docs/node/run-your-node/paratime-observer-node.mdx

```diff
@@ -0,0 +1,143 @@
+import DocCard from '@theme/DocCard';
+import {findSidebarItem} from '@site/src/sidebarUtils';
+
+# ParaTime Observer Node
+
+:::info
+
+These instructions are for setting up a _ParaTime observer_ node, which observes ParaTime activity and can issue
```

Comment:
Furthermore, I feel we have too many of these alert boxes that can just confuse newcomers. It would be better to list all types of nodes with short descriptions on a parent page, and then say "If this is not the type of node you are looking for, check other types of nodes <link-to-page>."

Reply:
All node types are already listed on the parent page: https://docs.oasis.io/node/run-your-node/.
I feel the same about alert boxes, but changing the style just here won't help much. We should change this everywhere because alert boxes are even more present on pages for other node types. Moreover, they are often seen in other parts of the docs, so a decision to reduce them would affect other pages too.